velox/exec/Window.h

```diff
@@ -65,6 +73,8 @@ class Window : public Operator {
     const core::WindowNode::WindowType type;
     const core::WindowNode::BoundType startType;
     const core::WindowNode::BoundType endType;
+    const WindowFrameFunctionPtr findStartFrame;
```

Comment:
Maybe call this startFrameFunction and endFrameFunction.

Reply:
Sure, thanks for the suggestion!